docusaurus/android_versioned_docs/version-5.x.x/04-compose/03-general-customization/02-attachment-factory.mdx

```diff
@@ -53,16 +53,41 @@ override fun onCreate(savedInstanceState: Bundle?) {
     setContent {
         // override the default factories by adding your own
         ChatTheme(attachmentFactories = myAttachmentFactories + defaultFactories) {
-            ChannelsScreen(...)
+            // Chat components
```

Comment:
Factories have an effect across multiple components so I figures a general example might be better here.